eoscostarica/guias.eoscostarica.io
# feat(frontend)GuíaGitHub-168

### GH Issue

Resolves #168  

### Steps to test
1. Go to tutoriales/Guía GitHub
2. Check content is right.

#### CheckList
- [ ] Under the tutorials section
